packages/chrome/src/styled/sheet/StyledSheet.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet';
+
+interface IStyledSheetProps {
+  placement: 'start' | 'end';
+}
+
+const sheetSmartBorderStyle = ({
+  theme,
+  placement
+}: IStyledSheetProps & ThemeProps<DefaultTheme>) => {
+  let borderSide = '';
+
+  if (placement === 'end') borderSide = '-left';
+  else if (placement === 'start') borderSide = '-right';
+
+  return `border${borderSide}: 1px solid ${theme.palette.grey[300]};`;
+};
+
+export const StyledSheet = styled.aside.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  tabIndex: -1
+})<IStyledSheetProps>`
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  order: 1;
+
+  transition: width 0.25s ease-in-out;
+  will-change: width;
```

Comment:
The property was intended to be a performance improvement for animations. Mike and I weren't aware it was intended to be used as a last resort. We'll remove it from the `StyledSheet` since there aren't any performance problems, to begin with. Thanks!

Reply:
Good call on this, will remove.